YarnSpinner/Value.cs

```diff
@@ -0,0 +1,334 @@
+
+using System;
+
+namespace Yarn
+{
+	// A value from inside Yarn.
+	public class Value : IComparable, IComparable<Value> {
```

Comment:
Once the overloaded operators started showing up it was almost mandatory. Commenting all the JS semantics take up a ton of space, haha.